src/fsharp/DotNetFrameworkDependencies.fs

```diff
@@ -82,7 +82,10 @@ module internal FSharp.Compiler.DotNetFrameworkDependencies
         let file =
             try
                 let depsJsonPath = Path.ChangeExtension(Assembly.GetEntryAssembly().Location, "deps.json")
-                File.ReadAllText(depsJsonPath)
+                if File.Exists(depsJsonPath) then
```

Comment:
@cartermp …
Lol … not throwing if the file does not exist improves the debugging experience by an immense amount.
The try … catch is for correctness … the if File.Exists is for my debugging pleasure.

I am going to put this change back into master branch too.  In fact, I should have ages ago, since most of what I do I add this when I get to debugging.